upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.6.yaml.template

```diff
@@ -237,30 +249,75 @@ spec:
 
 ---
 
-# This manifest deploys the Calico policy controller on Kubernetes.
-# See https://github.com/projectcalico/k8s-policy
+# This deployment turns off the old "policy-controller". It should remain at 0 replicas, and then
```

Comment:
Not sure how upgrade works, but we'll only need this here if it's possible to upgrade from a v1.6 cluster with Calico v2.4 to a cluster with Calico v2.6.

It won't hurt either way.

Reply:
Yeah should probably remove it.

Reply:
@blakebarnett we need to leave it, as an upgrade will not work without it.  We do a kubectl apply, which will not delete old stuff.

Reply:
no, I mean the new 2.6.2 version of `calico-kube-controllers` should be removed and we should run v2.4.x on k8s 1.6 with `calico-policy-controller` because it's the most widely tested configuration.

Reply:
We cannot run `calico-policy-controller` on a weave cluster.  It does not work.

Reply:
of course not... :)  This is only part of the calico addon.